docker/worker/worker.py

```diff
@@ -333,6 +333,10 @@ def _source_update(self, message):
     deleted = message.attributes['deleted'] == 'true'
 
     source_repo = osv.get_source_repository(source)
+    if source_repo is None:
+      logging.info('Failed to get source repository %s', source)
```

Comment:
Should this be logging.error instead? I don't think we should be expecting invalid sources.

Reply:
yes!